src/components/Sections/SectionFour.js

```diff
@@ -7,15 +7,15 @@ import Person from "../../../public/assets/person.png";
 
 const SectionFour = () =>  {
   return (
-    <div className="flex flex-col gap-4 justify-center items-center">
-        <div className="uppercase text-2xl font-semibold">
+    <div className="flex flex-col gap-2 justify-center items-center">
```

Comment:
Kindly add a default left and right margin here mx, 
![image](https://github.com/user-attachments/assets/dfaefe69-1947-4bba-8752-3511ffc11e09)
Wala ng spacing left on smaller devices